tests/contrib/cherrypy/test_middleware.py

```diff
@@ -121,7 +121,6 @@ def test_success(self):
         self.assertHeader("Content-Type", "text/html;charset=utf-8")
         self.assertBody("Hello world!")
 
-        assert not self.tracer.current_span(), self.tracer.current_span().pprint()
```

Comment:
oh no, we need to keep `assert not self.tracer.current_span()`, we can just drop the `, self.tracer.current_span().pprint()`

which actually... how the hell does this work? if there is no current span, then the assert message would also fail 🤔

Reply:
I'll fix this